contracts/TokenPool.sol

```diff
@@ -121,31 +121,31 @@ contract TokenPool is Migratable, Pausable {
 
         // copy permissions
 
-        bool oldIdIrnAdmin = atonomiStorage.getBool(keccak256("network", msg.sender, "isIRNAdmin"));
-        bool oldIsManufacturer = atonomiStorage.getBool(keccak256("network", msg.sender, "isManufacturer"));
-        bool oldIsIRNNode = atonomiStorage.getBool(keccak256("network", msg.sender, "isIRNNode"));
-        bytes32 oldMemberId = atonomiStorage.getBytes32(keccak256("network", msg.sender, "memberId"));
+        bool newIdIrnAdmin = atonomiStorage.getBool(keccak256("network", _new, "isIRNAdmin"));
+        bool newIsManufacturer = atonomiStorage.getBool(keccak256("network", _new, "isManufacturer"));
+        bool newIsIRNNode = atonomiStorage.getBool(keccak256("network", _new, "isIRNNode"));
+        bytes32 newMemberId = atonomiStorage.getBytes32(keccak256("network", _new, "memberId"));
 
-        require(!oldIdIrnAdmin, "already an irn admin");
-        require(!oldIsManufacturer, "already a manufacturer");
-        require(!oldIsIRNNode, "already an irn node");
-        require(oldMemberId == 0, "already assigned a member id");
+        require(!newIdIrnAdmin, "already an irn admin");
+        require(!newIsManufacturer, "already a manufacturer");
+        require(!newIsIRNNode, "already an irn node");
+        require(newMemberId == 0, "already assigned a member id");
 
 
-        atonomiStorage.setBool(keccak256("network", _new, "isIRNAdmin"), oldIdIrnAdmin);
-        atonomiStorage.setBool(keccak256("network", _new, "isManufacturer"), oldIsManufacturer);
-        atonomiStorage.setBool(keccak256("network", _new, "isIRNNode"), oldIsIRNNode);
-        atonomiStorage.setBytes32(keccak256("network", _new, "memberId"), oldMemberId);
+        atonomiStorage.setBool(keccak256("network", _new, "isIRNAdmin"), newIdIrnAdmin);
```

Comment:
This still doesnt look write.  the new account should be set from the old variables 

```
network[_new] = NetworkMember(
            old.isIRNAdmin,
            old.isManufacturer,
            old.isIRNNode,
            old.memberId
);
```

Reply:
newIdIrnAdmin should be replaced with oldIdIrnAdmin

Reply:
looks good now